### ISSUE TITLE
Raw line number

### ISSUE THREAD

**Issue by Yamakaky** [Marwes/combine#69]
Using the last example from https://docs.rs/combine/2.0.0/combine/,
```
fn main() {
    if let Err(e) = parser(expr).parse("[") {
        println!("{}", e);
    }
}
```
outputs
```
Parse error at 94334017221597
Unexpected end of input
Expected ]
```
instead of the beautiful `line: 1, column: 1` I was promised XD

**Comment by Marwes** [comment]
Oops, that has changed to require an explicit `State::new`
```
fn main() {
    if let Err(e) = parser(expr).parse(State::new("[")) {
        println!("{}", e);
    }
}
```

**Comment by Marwes** [comment]
With 2.0 I removed the implicit line + column tracking from parsing `&str` so to regain that behaviour one needs to explicitly wrap the input with `State` to get the old behaviour.

**Comment by Yamakaky** [comment]
Ok, cool!

**Comment by Marwes** [comment]
Fixed in efa2eb1cc8996eab1a29fd84c7219dcfcacf65c6 (guess it won't show up on docs.rs until I upload a new version though). Thanks!
